indra/llprimitive/llprimitive.cpp

```diff
@@ -1588,6 +1605,15 @@ S32 LLPrimitive::unpackTEMessage(LLDataPacker &dp)
         material_ids[i].set(&(material_data[i]));
     }
 
+    if ((cur_ptr < buffer_end) && (*cur_ptr == EXTRA_PROPERTY_ALPHA_GAMMA))
+    {
+        ++cur_ptr; // skip the indicator
+        if (!unpack_TEField<U8>(alpha_gamma, face_count, cur_ptr, buffer_end, MVT_U8))
+        {
+            LL_WARNS("TEXTUREENTRY") << "Malformed alphagamma unpacking texture entry." << LL_ENDL;
+        }
+    }
+
```

Comment:
Whoops, the version of `LLPrimitive::unpackTEMessage()` being used for the data from the server had not been instrumented to handle the new "AlphaGamma" data!  This is a good example of how bugs creep into code that is not **DRY**'ed.  Unfortunately, de-duplicating the code paths here would be a little harder.  It should be done, however I'm willing to be convinced we shouldn't block this PR for it.

Reply:
Disclaimer: popping in as I drive across the country for my move.

I see you've largely taken on de-crustifying things - though I don't feel like I'm the best person to really review that given my inexperience with the message system (for now - probably gonna be taking a few stabs with the upcoming VR work).  It seems like this comment is obsolete?

Reply:
Yes, this comment is obsolete.  I will resolve the conversation.